src/test/scala/com/databricks/spark/xml/util/XSDToSchemaSuite.scala

```diff
@@ -118,4 +116,11 @@ class XSDToSchemaSuite extends AnyFunSuite {
     assert(expectedSchema === parsedSchema)
   }
 
+  test("decimal-with-restriction") {
```

Comment:
Maybe a slightly more descriptive name, but no big deal

Reply:
Updated the test case name. Hope it's more descriptive